pkg/template/mixer/BUILD

```diff
@@ -0,0 +1,26 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_go//go:def.bzl", "go_prefix")
+load("@org_pubref_rules_protobuf//go:rules.bzl", "go_proto_library")
+
+go_proto_library(
```

Comment:
So, the code that does the parsing and generates the interface right now is using the parsing logic from the protobuf/protoc-gen-go. Since we have to compile the generated files along with the generated pb.go files, I think using the same generator from where we are using the parser will be safe and less chances of build break between our generated file and the tool's generated pb.go file.